### PR TITLE
Add type annotations to `wandb/env.py` and `wandb/sdk/internal/internal_api.py`

### PR DESCRIPTION
**Pull request by dmitryduev**
Description
-----------
First take on type-annotating `internal_api.py` in a self-consistent way. Once merged, we should plan to make annotations more specific for individual "units" of the code, one by one.

Increases type annotation coverage from 56.63% (current master) to 59.02%.

Testing
-------
How was this PR tested?

Checklist
-------
- Include reference to internal ticket "Fixes WB-NNNN" (and github issue "Fixes #NNNN" if applicable)

### CODE REVIEW

**Comment by dmitryduev** [comment]
Once this is merged, merge https://github.com/wandb/yea-wandb/pull/73

File: tox.ini [hunk 1]
```diff
@@ -91,14 +91,6 @@ usedevelop = true
 basepython = python3
 commands = ipython
 
-; [testenv:codemod]
```
Comment (kptkin) [inline tox.ini:94]: do you want to remove this?

File: tox.ini [hunk 2]
```diff
@@ -115,14 +107,6 @@ deps =
 commands=
     python ./tools/bumpversion-tool.py --from-dev {posargs}
 
-; [testenv:codemodcheck]
```
Comment (kptkin) [inline tox.ini:118]: same as above

File: tox.ini (hunk 1 above)
Comment (dmitryduev) [reply tox.ini:94]: yea, that's what I did, or?..

File: wandb/filesync/upload_job.py [hunk 3]
```diff
@@ -100,7 +100,8 @@ def push(self) -> bool:
             # This is the new artifact manifest upload flow, in which we create the
             # database entry for the manifest file before creating it. This is used for
             # artifact L0 files. Which now is only artifact_manifest.json
-            response = self._api.create_artifact_manifest(
+            # TODO: FOUND THIS BUG, IS THIS RELEVANT?
+            _, response = self._api.create_artifact_manifest(
```
Comment (kptkin) [inline wandb/filesync/upload_job.py:104]: wow... how does it work correctly?

Comment (dmitryduev) [reply wandb/filesync/upload_job.py:104]: we didn't have a test for this :(

**Review by kptkin — APPROVED** [review]
🔥

**Pull request merged**